extensions/amp-reach-player/amp-reach-player.md

```diff
@@ -20,13 +20,12 @@ limitations under the License.
 <table>
   <tr>
     <td width="40%"><strong>Description</strong></td>
-    <td>
-    An <code>amp-reach-player</code> component displays the Reach Player configured in the <a href="http://beachfrontreach.com">Beachfront Reach</a> platform.
+    <td>Displays the Reach Player configured in the <a href="http://beachfrontreach.com">Beachfront Reach</a> platform.
     </td>
   </tr>
   <tr>
     <td width="40%"><strong>Availability</strong></td>
-    <td>Beta</td>
+    <td>Stable</td>
```

Comment:
think the availability field is sort of redundant when it says "stable". I'd much prefer to remove it all together, and create a special callout for things that are not stable.